examples/bzlmod/cross_compile/.bazelrc

```diff
@@ -0,0 +1,85 @@
+# Instead, set a prefix and put it in .gitignore
+build --symlink_prefix=target-bzl
+
+# Non-empty glob test is disabled because some Bazels deps fail the test.
+# build --incompatible_disallow_empty_glob
+
+# This flag improves execution throughput by executing more actions in parallel.
+# build --jobs=32
+
+###############################################################################
+## Test configuration
+###############################################################################
+
+# Reduce test output to just error cases and report timeout warnings.
+test --test_output=errors
+test --verbose_failures
+
+###############################################################################
+## Common configuration
+###############################################################################
+
+# Enable Bzlmod for every Bazel command
+common --enable_bzlmod
+
+# Allow + in repo names for Bazel 8
+# common --incompatible_use_plus_in_repo_names
+
+# Write build outputs in a platform-specific directory;
+# avoid outputs being wiped and rewritten when switching between platforms.
+common --experimental_platform_in_output_dir
+
+# Enable proto toolchain resolution
+common --incompatible_enable_proto_toolchain_resolution
+
+# Enable misc. performance optimizations.
+# https://www.buildbuddy.io/blog/debugging-slow-bazel-builds/
+# https://www.buildbuddy.io/blog/how-bazel-7-0-makes-your-builds-faster/
+common --nolegacy_important_outputs
+common --verbose_failures
+common --noexperimental_merged_skyframe_analysis_execution
+common --reuse_sandbox_directories
+
+# Enable a more detailed performance profile
+common --noslim_profile
+common --experimental_profile_include_target_label
+common --experimental_profile_include_primary_output
+
+# Required for cargo_build_script support before Bazel 7
+common --incompatible_merge_fixed_and_default_shell_env
+
+###############################################################################
+## Cross compilation configuration
+###############################################################################
+
+# Debug toolchain resolution
+# build --toolchain_resolution_debug='@bazel_tools//tools/cpp:toolchain_type
+
+# Required to support cross compilation from different host systems i.e Mac and Linux
+# https://bazel.build/reference/command-line-reference#flag--enable_platform_specific_config
+common --enable_platform_specific_config
+
+# Required to resolve C/C++ toolchain.
+# https://github.com/bazel-contrib/musl-toolchain?tab=readme-ov-file#setup
+build --incompatible_enable_cc_toolchain_resolution
+
+###############################################################################
+## Custom user flags
+##
+## This should always be the last thing in the `.bazelrc` file to ensure
+## consistent behavior when setting flags in that file as `.bazelrc` files are
+## evaluated top to bottom.
+###############################################################################
+try-import %workspace%/user.bazelrc
```

Comment:
For the .blazerc files in in test subdirectories, it looks like we try to keep them tiny with only the settings necessary for the specific example to work, e.g.,: https://github.com/bazelbuild/rules_rust/blob/main/examples/bzlmod/hello_world/.bazelrc

Could you remove the settings that are not necessary?

Reply:
sure, let me clean this up